deb/default.json

```diff
@@ -692,7 +692,7 @@
 				"clone3"
 			],
 			"action": "SCMP_ACT_ERRNO",
-			"errnoRet": 38,
+			"errnoRet": 95,
 			"excludes": {
```

Comment:
Yeah, this was a bit too naive of course; I thought "let's try change the default", but that means missing syscalls (not defined in the profile) will probably get `ENOSYS` as well;

```
pkg-buildpackage: info: source changed by Docker <support@docker.com>
 dpkg-source -I.git --before-build .
dpkg-buildpackage: info: host architecture amd64
 debian/rules clean
dh clean --with=bash-completion
make: dpkg-parsechangelog: Operation not permitted
make: dh: Operation not permitted
make: *** [debian/rules:121: clean] Error 127
dpkg-buildpackage: error: debian/rules clean subprocess returned exit status 2
make[1]: *** [Makefile:85: ubuntu-jammy] Error 2
make[1]: Leaving directory '/home/runner/work/docker-ce-packaging/docker-ce-packaging/deb'
make: *** [Makefile:83: ubuntu-jammy] Error 2
Error: Process completed with exit code 2.
```